RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -43,6 +43,25 @@
     InitialStepPreSplitting +
     InitialStep +
     HighPtTripletStep +
+    DetachedQuadStep +
```

Comment:
@slava77 The `trackingPhase1PU70` is intentionally kept as the SLHC-tracking for reference, and will be cleaned up once we have a production-ready tracking for 2017 (at the latest). All future high-pileup developments should be based on run2/2017-production tracking.